pkg/testdatagen/scenario/e2ebasic.go

```
@@ -30,6 +30,23 @@ func (e e2eBasicScenario) Run(db *pop.Connection) {
 		},
 	})
 
+	// Service member with a new move
+	nowTime := time.Now()
+	ppm0 := testdatagen.MakePPM(db, testdatagen.Assertions{
+		ServiceMember: models.ServiceMember{
+			ID: uuid.FromStringOrNil("94ced723-fabc-42af-b9ee-87f8986bb5c9"),
+		},
+		Move: models.Move{
+			ID:      uuid.FromStringOrNil("0db80bd6-de75-439e-bf89-deaafa1d0dc8"),
+			Locator: "VGHEIS",
+		},
+		PersonallyProcuredMove: models.PersonallyProcuredMove{
+			PlannedMoveDate: &nowTime,
+		},
+	})
+	ppm0.Move.Submit()
+	save(db, &ppm0.Move)
```

I'd prefer to have these be more of a `mustSave` kind of thing. I agree that we don't need to handle these errors, but I'd rather have the program asplode than silently hide them.

---

Added private mustSave func to scenario pkg